### DISCUSSION TITLE
Steam only works when launched via the terminal!

### DISCUSSION THREAD

**Discussion by jbsbbruno** [canonical/steam-snap#393] [Q&A] [answered]
The software only starts its graphical interface when started by the terminal. The shortcut created by installation from the Ubuntu store only opens the notification icon. The first figure shows only the notification. The second figure shows Steam working, but only when launched by the terminal.

## Relatório de detalhes do sistema
---

## Detalhes do relatório
- **Data em que foi gerado:**                      2024-07-13 22:30:45

## Hardware Information:
- **Modelo do hardware:**                          Acer Aspire A315-23G
- **Memória:**                                     20,0 GiB
- **Processador:**                                 AMD Ryzen™ 5 3500U with Radeon™ Vega Mobile Gfx × 8
- **Gráficos:**                                    AMD Radeon™ Vega 8 Graphics
- **Gráficos: 1**                                  AMD Radeon™ 530 Series
- **Capacidade de disco:**                         768,2 GB

## Software Information:
- **Versão do firmware:**                          V1.21
- **Nome do SO:**                                  Ubuntu 24.04 LTS
- **Compilação do SO:**                            (null)
- **Tipo do SO:**                                  64 bits
- **Versão do GNOME:**                             46
- **Sistema de janelas:**                          Wayland
- **Verificação do kernel:**                       Linux 6.8.0-38-generic

![Captura de tela de 2024-07-13 22-19-23](https://github.com/user-attachments/assets/520883a0-e2da-49e9-ad31-cd56d37a070b)
![Captura de tela de 2024-07-13 22-21-48](https://github.com/user-attachments/assets/2518c63a-1d26-4086-b238-9b51f15d0262)

**Comment by ashuntu** [comment, marked as answer]
This should be an issue instead. However, there already exists an issue for this: https://github.com/canonical/steam-snap/issues/384. Closing since this is a duplicate.
